src/validator.ts

```diff
@@ -38,7 +38,7 @@ export class SurveyValidator extends Base {
   get locText(): LocalizableString {
     return this.getLocalizableString("text");
   }
-  protected getErrorText(name: string): string {
+  public getErrorText(name: string): string {
```

Comment:
Don't need to make it public you don't need to create the error, validator.validate()